docker_cli/cli/build_commands.py

```diff
@@ -39,6 +39,24 @@ def init_build_parsers(subparsers: argparse._SubParsersAction) -> None:
         help="The path to place the contents of the Git archive at on the image.",
     )
 
+    build_type_choices = ["Release", "Debug", "RelWithDebInfo", "MinSizeRel"]
+    config_params = argparse.ArgumentParser(add_help=False)
+    config_params.add_argument(
+        "--build-type",
+        type=str,
+        default="Release",
+        metavar="DCMAKE_BUILD_TYPE",
```

Comment:
Let's just call this "CMAKE_BUILD_TYPE". The "-D" part in the beginning is just a cmake command line flag -- it's not actually part of the variable name.

```suggestion
        metavar="CMAKE_BUILD_TYPE",
```